source/Backend/BackendInit.js

```diff
@@ -23,7 +18,16 @@ getUserID().then((user) => {
  * @param {String} dayObj.notes - a string representing the notes
  * @returns void
  */
-function createDay(dayObj) {
+async function createDay(dayObj) {
+    const currentUserID = await getUserID()
```

Comment:
Talked about adding this as a cookie, so we don't have call this getter every time. Perhaps in a next iteration